worldedit-core/src/main/java/com/boydti/fawe/object/extent/OffsetExtent.java

```diff
@@ -46,10 +43,4 @@ public boolean setBiome(int x, int y, int z, BiomeType biome) {
         throws WorldEditException {
         return getExtent().setBlock(x + dx, y + dy, z + dz, block);
     }
-
-    @Override
-    public ResettableExtent setExtent(Extent extent) {
-        mutable = new MutableBlockVector2();
-        return super.setExtent(extent);
-    }
```

Comment:
What was the reason for removing this?

Reply:
3D biomes means no need for the mutable vector so it doesn't need to be reset. Technically the extent doesn't even need to be resettable anymore so I'll change that as well.